File: problem-solving/A2SV/two-sum.py
class Solution:
    b=[1,1]
    n=1
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        Solution.n=len(nums)
        for i in range(Solution.n-1):
            for y in range(i+1,Solution.n,+1):
                if (nums[i]+nums[y])==target:
                    Solution.b[0]=i
                    Solution.b[1]=y
                    break
                
            else:
                pass
        return Solution.b

File: 1-two-sum/two-sum.go

func twoSum(nums []int, target int) []int {
    maps := make(map[int]int)
    for x,y := range nums{
        dif := target - y
        val,exist := maps[dif]
        if exist{
            ans := []int{x,val}
            return ans
        }
        maps[y] = x
    }
    ans := []int{1,1}
           
    return ans
    
}